README.md

```diff
@@ -37,7 +37,7 @@ PlatformPlatform is designed to support development on both Mac and Windows. The
 - [.NET](https://dotnet.microsoft.com)
 - [.NET Aspire](https://learn.microsoft.com/en-us/dotnet/aspire/) (run `dotnet workload update` and then `dotnet workload install aspire`)
 - [Docker Desktop](https://www.docker.com/products/docker-desktop/)
-- [Bun](https://bun.sh/docs/installation) (an all-in-one CLI tool for React development much faster than NodeJS - currently unstable on Windows)
+- [Npm](https://nodejs.org/en)
```

Comment:
Should this be Node? We don't use `npm` but `yarn`... but the prerequisites is Node, right?